Aliucord/src/main/java/com/aliucord/coreplugins/accountstanding/AccountStandingPage.kt

```diff
@@ -0,0 +1,191 @@
+/*
+ * This file is part of Aliucord, an Android Discord client mod.
+ * Copyright (c) 2025 Juby210 & Vendicated
+ * Licensed under the Open Software License version 3.0
+ */
+
+package com.aliucord.coreplugins.accountstanding
+
+import android.content.Context
+import android.graphics.drawable.GradientDrawable
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup.LayoutParams.MATCH_PARENT
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.*
+import androidx.core.content.res.ResourcesCompat
+import com.aliucord.*
+import com.aliucord.fragments.SettingsPage
+import com.aliucord.utils.GsonUtils
+import com.aliucord.utils.ViewUtils.addTo
+import com.discord.utilities.color.ColorCompat
+import com.lytefast.flexinput.R
+import com.aliucord.utils.DimenUtils.dp
+import com.aliucord.utils.SerializedName
+import com.discord.api.utcdatetime.UtcDateTime
+import com.discord.stores.StoreStream
+import com.discord.utilities.images.MGImages
+import com.facebook.drawee.view.SimpleDraweeView
+
+internal data class SafetyHubResponse(
+    @SerializedName("account_standing")
+    val accountStanding: AccountStandingState,
+    val classifications: List<UserClassifications>
+) {
+    data class UserClassifications(
+        val id: Long,
+        val description: String,
+        @SerializedName("flagged_content")
+        val flaggedContent: List<FlaggedContent>?,
+        val actions: List<Actions>?,
+        @SerializedName("max_expiration_time")
+        val maxExpirationTime: UtcDateTime
+    )
+
+    data class Actions(val descriptions: List<String>)
+    data class FlaggedContent(val content: String?)
+    data class AccountStandingState(val state: Int)
+
+    class DetermineString(val state: Int) {
```

Comment:
Noo don't make a useless class for this. Just add them directly to the AccountStandingState class